SymSettings.cfg

```diff
@@ -2,7 +2,7 @@
 # Global Settings
```

Comment:
You need to revert all the changes to SymSettings.cfg, remember to set things on the command line instead so that you don't accidentally commit changes to the config file